### PR TITLE
Implement Help and Version Commands 

### PR DESCRIPTION
**Pull request by s1ntaxe770r**
This PR adds version and help commands to the digger CLI
As requested in  #18

### CODE REVIEW

File: pkg/utils/commands.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	Name        string
+	Description string
+}
+
+var availableCommands = []Commad{
+	{"digger help", "Display help information"},
+	{"digger version", "Display version information"},
+	{"digger apply", "Apply digger configuration"},
+	{"digger plan", "Plan digger configuration"},
+	{"digger lock", "Lock Terraform state"},
```
Comment (Spartakovic) [inline pkg/utils/commands.go]: there is digger unlock too. Also it doesn't lock terraform state, it rather locks terraform projects on pull request level

File: pkg/utils/commands.go [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+type Commad struct {
+	Name        string
+	Description string
+}
+
+var availableCommands = []Commad{
+	{"digger help", "Display help information"},
+	{"digger version", "Display version information"},
+	{"digger apply", "Apply digger configuration"},
+	{"digger plan", "Plan digger configuration"},
```
Comment (Spartakovic) [inline pkg/utils/commands.go]: it doesn't apply digger configuration, but terraform configuration. It just adds few more things on top, like pull request locking

**Comment by motatoes** [comment]
In addition to being invoked as a cli command, digger is also invoked as part of github action in which case the arguments are github context. We also want to react to a comment in the pull request and publish a comment in result. We can do this [here](https://github.com/diggerhq/digger/blob/3fffbdcc2073de471f92e2f3a634b251008cbdd0/cmd/digger/main.go#L59-L58) after detecting that the event is a comment and the comment is "digger help" then publish a comment as a result to it and exit(0)

File: cmd/digger/main.go [hunk 3]
```diff
@@ -57,6 +67,12 @@ func main() {
 	}
 	println("GitHub event processed successfully")
 
+	if eventName == "pull_request" && parsedGhContext.Action == "commented" {
+		reply := utils.GetCommands()
+		githubPrService.ReplyComment(prNumber, reply)
+		os.Exit(0)
+	}
```
Comment (motatoes) [inline cmd/digger/main.go]: I suggest to do it like this

```suggestion
	if digger.CheckIfHelpComment(ghEvent) {
		reply := utils.GetCommands()
		githubPrService.PublishComment(prNumber, reply)
	}
```

Where you can define the function of `CheckIfHelpComment` as follows in digger.go

```
func CheckIfHelpComment(event models.Event) bool {
	switch event.(type) {
	case models.IssueCommentEvent:
		event := event.(models.IssueCommentEvent)
		if strings.Contains(event.Comment.Body, "digger help") {
			return true
		}
	}
	return false
}
```

File: pkg/github/github.go [hunk 4]
```diff
@@ -24,6 +25,7 @@ type GithubPullRequestService struct {
 type PullRequestManager interface {
 	GetChangedFiles(prNumber int) ([]string, error)
 	PublishComment(prNumber int, comment string)
+	ReplyComment(prNumber int, reply string)
```
Comment (motatoes) [inline pkg/github/github.go]: This function is not needed at all

**Review by motatoes — CHANGES_REQUESTED** [review]
few last changes, but looking good so far

**Review by motatoes — APPROVED** [review]

**Pull request merged**